eo-runtime/src/test/groovy/check-folders-numbering.groovy

```diff
@@ -0,0 +1,40 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2023 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation directories (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+target = basedir.toPath().resolve("target").resolve("eo")
+def directories = target.toFile().listFiles(new FileFilter() {
+  @Override
+  boolean accept(final File pathname) {
+    return pathname.isDirectory()
+  }
+})
+assert 6 == directories.size()
+[
+  target.resolve('1-parse').toFile(),
+  target.resolve('2-optimize').toFile(),
+  target.resolve('3-pull').toFile(),
+  target.resolve('4-resolve').toFile(),
+  target.resolve('5-pre').toFile(),
+  target.resolve('6-transpile').toFile(),
+].each { assert directories.contains(it) }
```

Comment:
@volodya-lombrozo if tomorrow we will have `5-something` directory there, your test won't detect it, right?

Reply:
@yegor256 I've updated test logic little bit in order to catch case with `5-something`. Could you take a look again, please?